### DISCUSSION TITLE
Can't get shared business folder to sync

### DISCUSSION THREAD

**Discussion by Samuel3k** [abraunegg/onedrive#1625] [Q&A] [answered]
I'm trying to get a shared business folder to sync, but when onedrive is trying to sync the shared business folder, it seems to just skip over it. Is there anything I'm missing? I've attached screenshots of my configuration.
![2021-09-08_21-47](https://user-images.githubusercontent.com/53394877/132577140-b7d41e94-5d0a-4429-8e8a-f744865f2c9c.png)
![2021-09-08_21-46](https://user-images.githubusercontent.com/53394877/132577144-556e2d4a-f491-4f6d-9def-70f7d964e496.png)
![2021-09-08_21-51](https://user-images.githubusercontent.com/53394877/132577154-0781ba0e-4cc6-4801-8e15-ff603dd5eb9f.png)

**Comment by abraunegg** [comment, marked as answer]
@Samuel3k 
> I'm trying to get a shared business folder to sync, but when onedrive is trying to sync the shared business folder, it seems to just skip over it. Is there anything I'm missing?

**Edit:** You are using an unsupported client version (v2.4.10) - you must upgrade to v2.4.13 or greater.

How are you starting the client? What command are you using? Are you passing in `--sync-shared-folders` or have you configured `sync_business_shared_folders = "true"` in your config file?

* https://github.com/abraunegg/onedrive/blob/master/docs/BusinessSharedFolders.md#performing-a-sync-of-onedrive-business-shared-folders
* https://github.com/abraunegg/onedrive/blob/master/docs/BusinessSharedFolders.md#enable-syncing-of-onedrive-business-shared-folders-via-config-file

**Reply by Samuel3k** [reply]
I upgraded my client and enabled the syncing of shared folders in my config. That seems to have solved the issue. Thank you for your answer.
